ubg/src/pages/Game.js

```diff
@@ -250,17 +250,21 @@ function Description({usersCollection, game, createRoom}) {
  */
 function FavoriteButton({usersCollection, game}) {
   const user = useUser();
-  const userGames = useFirestoreDocData(usersCollection.doc(user.uid)).games;
+  const userGames = useFirestoreDocData(
+      usersCollection.doc((user && user.uid) || ' ')).games;
```

Comment:
Do you need user && user.uid? Or can you just use user.uid?

Reply:
I need that there because it'll fail trying to read user.uid

Reply:
Yes, but if just have user.uid there and its null then its going to go with '  '

Reply:
![Screenshot 2020-08-05 at 11 54 42 AM](https://user-images.githubusercontent.com/23410540/89441266-7e812f00-d712-11ea-81c2-2ab0c0de5faf.png)

Reply:
Something like that, I mean its just so you don't have to do the && it's just to get rid of one extra condition.

Reply:
the issue is user is null already, so trying to access the uid of a null object causes problems.